theano/gof/utils.py

```diff
@@ -221,7 +221,7 @@ def __call__(cls, *args, **kwargs):
         for arg in args:
             arg = make_hashable(arg)
             key.append(arg)
-        for k, v in kwargs.iteritems():
+        for k, v in kwargs.items():
```

Comment:
We had an error here in python 3. So I changed it to ```items()``` which is supported in both.